src/schema/submission.yaml

```diff
@@ -1156,12 +1186,14 @@ slots:
     description: >-
       Data use permission associated with an entity. Typically one or more terms from DUO.
```

Comment:
Data use permission is not multivalued. Same as the previous comments on this slot.